Changelog.md

```diff
@@ -4,7 +4,7 @@ Language Features:
 
 
 Compiler Features:
-
+* Optimized constant generation for masks. With low compiler runs (optimized for size), reduces the byte code length and gas.
```

Comment:
```suggestion
* Constant Optimizer: Compute masks using shifts when optimizing for size.
```
Feel free to adjust the bit after the colon, but to keep things consistent, we usually categorize by component (see e.g. 0.8.27 for reference)